db/migrate/20201204164913_add_contacts_nhs_number_unique.rb

```diff
@@ -0,0 +1,11 @@
+class AddContactsNhsNumberUnique < ActiveRecord::Migration[6.0]
+  disable_ddl_transaction!
+
+  def up
+    safety_assured { execute 'create unique index index_contacts_on_lower_nhs_number on contacts using btree (lower(nhs_number))' }
```

Comment:
I don't think this will work as there are already duplicates in the database. Not sure what best way forward is - maybe we just add validations at code level rather than database?

Reply:
Yes, we can add only model validations. But we will not be able to edit contacts with duplicate `nhs_number`. Can we:
1 - validate we have there are already duplicates in the database on production
2 - if yes, to ask what to do with them
?

Reply:
If there are duplicates for the same contact could you get away with adding a scope? https://guides.rubyonrails.org/active_record_validations.html#uniqueness

Reply:
Thanks @dmgarland we can use it for DOB